spec/spec_helper.rb

```diff
@@ -9,4 +9,12 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  config.disable_monkey_patching!
```

Comment:
This method name is terrifying compared to what the docs says it actually does...